Call curl with -k (--insecure) so it does not attempt to validate SSL certificates.
--- a/usr/lib/site_ruby/aes/amiutil/http.rb     Wed Feb 13 16:55:29 2008 -0500
+++ b/usr/lib/site_ruby/aes/amiutil/http.rb     Wed Feb 13 16:57:12 2008 -0500
@@ -58,7 +58,7 @@ module HTTP
     tf.close(false)
     
     begin
-      cmd_line = "curl -f  #{curl_arguments} #{url} 2> #{tf.path} | tee #{path} | openssl sha1; exit ${PIPESTATUS[0]}"
+      cmd_line = "curl -k -f  #{curl_arguments} #{url} 2> #{tf.path} | tee #{path} | openssl sha1; exit ${PIPESTATUS[0]}"
       calculated_digest = IO.popen( cmd_line ) { |io| io.readline.chomp }
       
       unless $?.exitstatus == 0
@@ -117,7 +117,7 @@ module HTTP
       curl_arguments = ""
     end
         
-    cmd_line = "curl -f -s #{curl_arguments} #{url} -T #{path}"
+    cmd_line = "curl -k -f -s #{curl_arguments} #{url} -T #{path}"
     system( cmd_line )
     
     unless $?.exitstatus == 0
@@ -145,7 +145,7 @@ module HTTP
       curl_arguments = ""
     end
     
-    cmd_line = "curl -f -s -X DELETE #{curl_arguments} #{url}"
+    cmd_line = "curl -k -f -s -X DELETE #{curl_arguments} #{url}"
     # @log.info { "HTTP curl command line: #{cmd_line}" }
     retval = system( cmd_line )
     raise Error::Transfer.new( "delete failed for #{url}") unless retval
